### DISCUSSION TITLE
Can a rule for Transaction A impact Transaction B?

### DISCUSSION THREAD

**Discussion by Terry-JF** [firefly-iii/firefly-iii#9062] [Firefly III] [answered]
I have a problem with importing transactions via GoCardless. The bank for my main Asset Account gives me notice of pending transactions that will hit my account on the next banking day. During an import these create transactions with a tag = "pending" and in a subsequent import create duplicate transactions with a tag = "booked".

Adding a rule that simply ignores/deletes "pending" transactions seems to be my only option unless there is a way that the "booked" transaction creation rule can update or delete the prior "pending" transaction so as to avoid creating a duplicate entry?

**Comment by JC5** [comment, marked as answer]
It is the easiest to delete transactions with the tag "pending". Firefly III will not attempt to merge them.

**Reply by Terry-JF** [reply]
Thanks James, I thought that might be the case but always worth checking.
